src/templates/zh-TW/2020/methodology.html

```diff
@@ -0,0 +1,476 @@
+          </ul>
+        </li>
+        <li><a href="#tools">工具</a>
+          <ul>
+            <li><a href="#webpagetest">WebPageTest</a></li>
+            <li><a href="#lighthouse">Lighthouse</a></li>
+            <li><a href="#wappalyzer">Wappalyzer</a></li>
+            <li><a href="#chrome-ux-report">Chrome UX Report</a></li>
+            <li><a href="#third-party-web">Third Party Web</a></li>
+            <li><a href="#rework-css">Rework CSS</a>
+              <ul>
+                <li><a href="#rework-utils">Rework Utils</a></li>
+              </ul>
+            </li>
+            <li><a href="#parsel">Parsel</a></li>
+          </ul>
+        </li>
+        <li><a href="#process">分析流程</a>
+          <ul>
+            <li><a href="#planning">規劃</a></li>
+            <li><a href="#analysis">分析</a></li>
+            <li><a href="#interpretation">說明</a></li>
+          </ul>
+        </li>
+        <li><a href="#looking-ahead">展望未來</a></li>
+      </ul>
+{% endblock %}
+
+{% block main_content %}
+    <section>
+      <!-- Show large image for large screens and high density screens and use webp when supported -->
+      <picture>
+          <source media="(min-width: 327px)" type="image/webp" srcset="/static/images/methodology-banner.webp" />
+          <source media="(min-width: 327px)" type="image/jpeg" srcset="/static/images/methodology-banner.png" />
+          <source type="image/webp" srcset="/static/images/methodology-banner.webp 2x" />
+          <source type="image/jpeg" srcset="/static/images/methodology-banner.png 2x" />
+          <source type="image/webp" srcset="/static/images/methodology-banner-sm.webp" />
+          <source type="image/jpeg" srcset="/static/images/methodology-banner-sm.png" />
+          <img src="/static/images/methodology-banner.png" class="content-banner" alt="" width="1200" height="984" loading="eager" />
+        </picture>
+        <h2 id="overview"><a href="#overview" class="anchor-link">總覽</a></h2>
+
+        <p>
+          Web Almanac 網路年鑑是由<a hreflang="en" href="https://httparchive.org"> HTTP Archive </a>組織的專案。 HTTP Archive 在2010年由 Steve Souders 創辦，任務目標為追蹤網路如何建構。它按月評估數以百萬的網頁並將 TB 位元數據置<a hreflang="en" href="https://httparchive.org/faq#how-do-i-use-bigquery-to-write-custom-queries-over-the-data"> BigQuery </a>供大眾分析。
+        </p>
+
+        <p>
+          Web Almanac 網路年鑑的任務是將網路狀態成為公開的年度知識庫。我們的目標是透過主題專家的洞察語意讓網路社群無障礙的進入 HTTP Archive 資料庫。
+        </p> 
+
+        <p>
+          Web Almanac 網路年鑑2020年版共有四個部分：內文、體驗、發佈和傳遞。在這當中，每個章節都透過不同角度探索相關主題，例如第二部分探索使用者經驗在效能、隱私、無障礙網頁等各種角度。
+        </p>
+    </section>
+
+    <section>
+        <h2 id="dataset"><a href="#dataset" class="anchor-link">關於資料集</a></h2>
+
+        <p>
+          HTTP Archive 資料集每月持續更新數據，除各章節另有說明外，2020年版所有指標都來自2020年8月。這些結果可以透過<a hreflang="en" href="https://github.com/HTTPArchive/httparchive.org/blob/master/docs/gettingstarted_bigquery.md">設置 BigQuery </a>公共資料集的相關日期探索查詢，例如<code>2020_08_01</code>。
```

Comment:
Do we need this extra space as part of the link?

```suggestion
          HTTP Archive 資料集每月持續更新數據，除各章節另有說明外，2020年版所有指標都來自2020年8月。這些結果可以透過<a hreflang="en" href="https://github.com/HTTPArchive/httparchive.org/blob/master/docs/gettingstarted_bigquery.md">設置 BigQuery</a>公共資料集的相關日期探索查詢，例如<code>2020_08_01</code>。
```

Reply:
OK then, yeah I think space outside the link is better. Unless you or @CYBAI disagree?